docs/customise-authentication-pages/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: "doc"
+title: "Customise authentication pages"
+abstract: "Add custom messaging and color schemes to the DID.app sign in."
+---
+
+*Upgrade to our startup tier, or higher, to start customising your sign in pages.*
+
+To edit the customisation options for an app, visit the admin page for that app.
+You will see the following options if there are available.
+
+PICTURE of options
+
+You can set a color to match your brand color scheme, this will be usef on all the buttons and highlights in the sign in widget.
```

Comment:
```suggestion
You can set a colour to match your brand colour scheme, this will be used on all the buttons and highlights in the sign in widget.
```